### PR TITLE
Partial #1559 #7

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/test/pcgen/core/CampaignTest.java [hunk 1]
```diff
@@ -38,7 +38,6 @@
 /**
  * <code>CampaignTest</code> is ...
  * 
- * Last Editor: $Author$ Last Edited: $Date: 2008-11-11 18:34:19 -0500
  * (Tue, 11 Nov 2008) $
```
Comment (thpr) [inline code/src/test/pcgen/core/CampaignTest.java]: This wrapped

File: code/src/test/plugin/jepcommands/ClassLevelCommandTest.java [hunk 2]
```diff
@@ -46,7 +46,6 @@
 /**
  * <code>OrCommandTest</code> tests the functioning of the jep or plugin
  * 
- * Last Editor: $Author: nuance $ Last Edited: $Date: 2007-10-26 22:19:47 -0400
  * (Fri, 26 Oct 2007) $
```
Comment (thpr) [inline code/src/test/plugin/jepcommands/ClassLevelCommandTest.java]: Wrapping here as well to be cleaned up

File: code/src/test/pcgen/core/CampaignTest.java (hunk 1 above)
Comment (grimreaper) [inline code/src/test/pcgen/core/CampaignTest.java]: can I do those as a separate PR? I'd prefer to keep this one entirely automated.
